### PR TITLE
Migrate 3D_image_classification tutorial to Keras(all backends)

### PR DESCRIPTION
**Pull request by SuryanarayanaY**
Migrated `'3D_image_classification'` tutorial from keras2 to Keras3.

The tutorial has tensorflow dependency for data preprocessing due to `tf.data`.

Tested the code with `'torch'` backend and it is executing fine as per attached [gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/a7b2fe08bff0b7205aac790b4e942abd/3d_image_classification_keras3-torch-success.ipynb).

Attached git diff file attached below.
[git_diff_3D-image-classification.txt](https://github.com/keras-team/keras-io/files/13897317/git_diff_3D-image-classification.txt)

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
LGTM, thank you! Please add the generated files.

**Comment by SuryanarayanaY** [comment]
@fchollet , Added generated files also. May please reveiw.

File: examples/vision/md/3D_image_classification.md [hunk 1]
```diff
@@ -423,46 +1504,111 @@ model.summary()
 <div class="k-default-codeblock">
 ```
 Model: "3dcnn"
+
 _________________________________________________________________
-Layer (type)                 Output Shape              Param #   
+
+ Layer (type)                Output Shape              Param #
```
[the remark this answers is not in the document]
Comment (SuryanarayanaY) [reply examples/vision/md/3D_image_classification.md]: My bad. Generated docs with Keras3.

**Comment by SuryanarayanaY** [comment]
@fchollet , Added generated files.

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**